palace/linalg/chebyshev.hpp

```diff
@@ -1,8 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-#ifndef PALACE_CHEBYSHEV_SMOOTHER_HPP
-#define PALACE_CHEBYSHEV_SMOOTHER_HPP
+#ifndef PALACE_LINALG_CHEBYSHEV_SMOOTHER_HPP
+#define PALACE_LINALG_CHEBYSHEV_SMOOTHER_HPP
```

Comment:
Could drop the `SMOOTHER`, doesn't match the file name unlike all the others.

Reply:
The include guards match the class name, not file name in this case.